src/boot/main.c

```diff
@@ -22,6 +22,9 @@
 #include "game/puppyprint.h"
 #include "game/profiling.h"
 #include "game/emutest.h"
+#ifdef F3DEX_GBI_3
+#include "game/f3dex3.h"
+#endif
```

Comment:
I would move the ifdef to the actual f3dex3 header rather than for each individual f3dex3 include